src/pages/JobLogs.tsx

```diff
@@ -0,0 +1,162 @@
+    groupId: string;
+  }>();
+  const execution = parseInt(execStr, 10);
+  const {
+    data: displayTaskResult,
+    loading: isLoadingDisplayTask,
+    error: errorDisplayTask,
+  } = useQuery<GetDisplayTaskQuery, GetDisplayTaskQueryVariables>(
+    GET_DISPLAY_TASK,
+    {
+      variables: { taskId, execution },
+      onError: (err) =>
+        dispatchToast.error(
+          `There was an error loading the task: ${err.message}`
+        ),
+    }
+  );
+  const [
+    getTests,
+    { data: testsResult, loading: isLoadingTests, error: errorTests },
+  ] = useLazyQuery<GetTestsQuery, GetTestsQueryVariables>(GET_TESTS, {
+    onError: (err) =>
+      dispatchToast.error(
+        `There was an error loading test result data: ${err.message}`
+      ),
+  });
+
+  const task = displayTaskResult?.task?.displayTask ?? displayTaskResult?.task;
+  useEffect(() => {
+    if (task) {
+      getTests({
+        variables: {
+          taskId: task.id,
+          execution: task.execution,
+          ...(groupId && { groupId }),
+        },
+      });
+    }
+  }, [task, groupId, getTests]);
+
+  if (errorTests || errorDisplayTask) {
+    return <PageDoesNotExist />;
+  }
+
+  const { testResults } = testsResult?.taskTests ?? {};
+  return isLoadingDisplayTask || isLoadingTests ? (
+    <Skeleton paragraph={{ rows: 8 }} />
+  ) : (
+    <Row>
+      <Card>
+        <Column>
+          <>
+            <H2>
+              <StyledRouterLink to={getTaskRoute(task?.id)} data-cy="task-link">
+                {task?.id}
+              </StyledRouterLink>
+            </H2>
+            <SubtitleContainer>
+              <Subtitle>
+                Execution: <span data-cy="execution">{task?.execution}</span>
+                <br />
+                {groupId && (
+                  <>
+                    Job Number: <span data-cy="groupId">{groupId}</span>
+                  </>
+                )}
+              </Subtitle>
+            </SubtitleContainer>
+          </>
+          {testResults?.length &&
+          !displayTaskResult?.task.executionTasks?.length ? (
+            <SubtitleContainer>
+              <Subtitle>
+                <StyledLink
+                  data-cy="complete-test-logs-link"
+                  onClick={() => {
+                    sendEvent({ name: "Clicked complete logs link", taskId });
+                  }}
+                  href={getLobsterTestLogCompleteUrl({
+                    taskId,
+                    groupId,
+                    execution,
+                  })}
+                >
+                  Complete logs for all
+                </StyledLink>
+              </Subtitle>
+            </SubtitleContainer>
+          ) : null}
+          {testResults?.map(
+            ({ id, lineNum, displayTestName, testFile, ...test }) => (
+              <StyledLink
+                href={getLobsterTestLogUrl({
+                  taskId: test.taskId,
+                  execution: test.execution,
```

Comment:
[nit] Seems like `test` could be destructured too
```suggestion
            ({ id, lineNum, displayTestName, testFile, taskId, execution }) => (
              <StyledLink
                href={getLobsterTestLogUrl({
                  taskId,
                  execution,
```

Reply:
this would be ideal but taskId and execution are being used in the outside scope